build/templates/grpc_session_options.py.mako

```diff
@@ -11,6 +11,10 @@ from enum import IntEnum
 # Customers can pass this value to the MeasurementLink discovery service to resolve the server instance that provides this interface.
 GRPC_SERVICE_INTERFACE_NAME = '${config["module_name"]}_grpc.${config["grpc_service_class_prefix"]}'
 
+# This constant specifies the API license key required by the NI gRPC Device Server that comes with
+# MeasurementLink 2023 Q1.
+MEASUREMENTLINK_23Q1_NIMI_PYTHON_API_KEY = 'DE10751B-3EE0-44EC-A93B-800E6A3C89E4'
```

Comment:
I read a little about API Keys and [documentation](https://cloud.google.com/endpoints/docs/grpc/when-why-api-key#security_of_api_keys) said that they're for identifying the project that's making the call. What's to stop this key from being used by other projects? Do we need to take steps to prevent that?

Reply:
No, not at this time. We might shore that up in a future release, and then deprecate the old key after a release or two has gone by.